docs/elasticsearch-specification.asciidoc

```diff
@@ -258,12 +258,46 @@ spec:
         secretName: my-cert
```

Comment:
Is it worth updating this to `quickstart-es-cert` so that someone can just combine both the ES spec example and the cert manager example?